### PR TITLE
Mr 641 load placemarks

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md
- [ ] This change has been tested via the terms in TESTING.md

### CODE REVIEW

File: cypress/integration/control-tags.spec.ts [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-  it("should work", () => {
+describe("Control Tags", () => {
+  it("control tags should be hidden by default", () => {
     cy.visit("/cypress/control-tags");
 
     cy.get(".meridian-map-container").should("exist");
     cy.get(".meridian-floor-label").should(
       "contain",
       "Main Building – Floor 01"
     );
```
Comment (vercasson) [inline cypress/integration/control-tags.spec.ts:11]: I reversed the assertions here to test default functionality first.

Comment (vercasson) [reply cypress/integration/control-tags.spec.ts:11]: and I think I might revert this

File: cypress/integration/load-placemarks.spec.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "Main Building – Floor 01"
+    );
+    cy.get("[data-meridian-placemark-id]").should("have.length", 31);
+
+    cy.window().then((contentWindow: any) => {
+      const meridianMap: MeridianMap = contentWindow.meridianMap;
+      meridianMap.update({
+        loadPlacemarks: false
+      });
+    });
```
Comment (wavebeem) [inline cypress/integration/load-placemarks.spec.ts:19]: Can you update `getMeridianMap` to be a util so we don't keep copy/pasting this?

File: cypress/integration/basic.spec.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
-describe("Example", () => {
-  it("should work", () => {
+describe("Basic/Default test", () => {
+  it("should load data and UI elements", () => {
```
Comment (wavebeem) [inline cypress/integration/basic.spec.ts:2]: I think `Basic Example` and `should load correctly` should be sufficient personally

File: cypress/integration/load-placemarks.spec.ts (hunk 2 above)
Comment (vercasson) [reply cypress/integration/load-placemarks.spec.ts:19]: 👍 Sure, I'll do that after Tara merges her PR where it becomes a util.

File: cypress/integration/control-tags.spec.ts (hunk 1 above)
Comment (vercasson) [reply cypress/integration/control-tags.spec.ts:11]: I did revert

File: cypress/integration/basic.spec.ts (hunk 3 above)
Comment (vercasson) [reply cypress/integration/basic.spec.ts]: I re-organized the assertions and broke some out into other tests.

File: cypress/integration/control-tags.spec.ts (hunk 1 above)
Comment (vercasson) [reply cypress/integration/control-tags.spec.ts:11]: I can explain tomorrow

File: cypress/integration/basic.spec.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-describe("Basic/Default test", () => {
+describe("Default tests", () => {
```
Comment (wavebeem) [inline cypress/integration/basic.spec.ts:1]: It's not a huge deal but my point was that if the example is called "basic" in the code I feel like we should call it "basic" in the describe block took, and putting "tests" in a describe block feels redundant to me

File: cypress/integration/basic.spec.ts [hunk 5]
```diff
@@ -24,10 +24,14 @@ describe("Default tests", () => {
...
-    cy.get("[data-meridian-tag-id]").should("have.length", 3);
+    cy.get("[data-meridian-tag-id]")
+      .should("have.length", 3)
+      .should("not.be.disabled");
 
     cy.get('[data-meridian-placemark-id="5653164804014080"]').should("exist");
-    cy.get("[data-meridian-placemark-id]").should("have.length", 31);
+    cy.get("[data-meridian-placemark-id]")
+      .should("have.length", 31)
+      .should("not.be.disabled");
```
Comment (wavebeem) [inline cypress/integration/basic.spec.ts:34]: Nice update

**Review by wavebeem — APPROVED** [review]

**Pull request merged**